src/state/slices/portfolioSlice/portfolioSliceCommon.ts

```diff
@@ -88,34 +88,8 @@ export type AccountMetadataById = {
   [k: AccountId]: AccountMetadata
 }
 
-/**
- * important note about this type
- *
- * for EVM and CosmosSDK-based chains, the address is the same as the account
- * for UTXO based chains, the account is an xpub or variety thereof, and addresses
- * belong to accounts.
- *
- * in the future we will refer to accounts on all varieties of chains as a Pubkey
- *
- * w.r.t this type, for
- * * Bitcoin Legacy this is a 1-prefixed address
- * * Bitcoin Segwit this is a 3-prefixed address
- * * Bitcoin Segwit Native this is a bc1-prefixed address
- *
- * as soon as we change websocket subscription logic, this entire byId section
- * can be removed from the store, as the only reason we currently maintain this mapping
- * is to be able to lookup an account from an address for websocket messages for UTXO chains
- */
-type Address = string
-
 export type PortfolioAccountSpecifiers = {
   accountMetadataById: AccountMetadataById
-  byId: {
```

Comment:
we had this whole mapping of address to account, only so we could upsert txs into the correct accounts based on the address over websockets.

now we subscribe by accountid we don't have this problem